.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: 💫 Feature request
+about: Ask us to implement something new
+---
+
+<!--- Verify first that your feature was not already discussed on GitHub -->
+<!--- Complete *all* sections as described -->
+
+# FEATURE
+
+## Summary
+
+Main idea
+
+### Component
+
+<!-- In which component this feature should be implemented -->
```

Comment:
I don't think it's necessary. Even core developers often don't know the component scope of a task in advance. Most tasks reqiure touching more than one component. And I don't see how filling this section saves us any time.

Reply:
Removed